### PR TITLE
Fix acquisition rate bkprofocusoem

### PR DESCRIPTION
**Pull request by tdowrick**
RE: https://github.com/PlusToolkit/PlusLib/issues/486#issuecomment-456881448

I have addressed the bug where the acquisition rate for the BK5000 was hardcoded, rather than using the rate specified in the config file, and tested successfully on our BK5000.

### CODE REVIEW

File: src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.cxx [hunk 1]
```diff
@@ -1019,6 +1031,7 @@ PlusStatus vtkPlusBkProFocusOemVideoSource::ReadConfiguration(vtkXMLDataElement*
   XML_READ_SCALAR_ATTRIBUTE_REQUIRED(int, OemPort, deviceConfig);
   XML_READ_BOOL_ATTRIBUTE_OPTIONAL(ContinuousStreamingEnabled, deviceConfig);
   XML_READ_BOOL_ATTRIBUTE_OPTIONAL(ColorEnabled, deviceConfig);
+  XML_READ_SCALAR_ATTRIBUTE_OPTIONAL(double, AcquisitionRate, deviceConfig);
```
Comment (adamrankin) [inline src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.cxx]: I believe this is handled by the base class ReadConfiguration?

File: src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.h [hunk 2]
```diff
@@ -47,6 +47,12 @@ class vtkPlusDataCollectionExport vtkPlusBkProFocusOemVideoSource : public vtkPl
   /*! BK scanner OEM port */
   vtkSetMacro(OemPort, unsigned short);
 
+  /*! BK scanner Acquisition Rate */
+  vtkSetMacro(AcquisitionRate, double);
+
+  /*! BK scanner Acquisition Rate */
+  vtkGetMacro(AcquisitionRate, double);
```
Comment (adamrankin) [inline src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.h]: Are these functions present in the base class?

File: src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.cxx (hunk 1 above)
Comment (adamrankin) [reply src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.cxx]: Yes, and the base class read configuration is handled by the `XML_FIND_DEVICE_ELEMENT_REQUIRED_FOR_READING` macro. This isn't needed.

File: src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.h (hunk 2 above)
Comment (adamrankin) [reply src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.h]: Confirmed they are, please remove.

**Comment by tdowrick** [comment]
Hi Adam, thanks for reviewing. I have made the requested changes.

**Pull request merged**